compiler/src/dmd/glue.d

```diff
@@ -69,8 +71,11 @@ import dmd.toobj;
 import dmd.typesem;
 import dmd.utils;
 
+private:
```

Comment:
This private doesn't do anything, except to attach itself to the following declaration. All other declarations seem to already be private or public.

I think that adding access specifiers or attributes en masse is a bad technique as the declaration site is no longer fully informative. I have been bitten by this in the past, so please, I suggest we stop using them.

On a side note: D has a lot of features that makes it easy to fix some issues at the cost at making the code less readable. Examples are: en-masse attributes/access specifiers, mixins, alias this etc. These features help the programmer to get an easy way out from some situations, but IMO have a bad effect on the long term maintenance of the code.

Reply:
> This private doesn't do anything, except to attach itself to the following declaration. All other declarations seem to already be private or public.

What it does is inform the reader that the public functions will be marked as `public` rather than defaulting to being public. Then, I can grep for the module's public interface. Over the years I've spent a lot of accumulated time wondering if an unmarked function was public or private.